main.py

```diff
@@ -353,49 +357,81 @@ async def on_message(self, message):
+        # Debug name
+        if debugMode: logger.debug
+
+        try:
+            cursor.execute(f"SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = '{name}'")
+        except Error as e:
+            logger.error(f"{e}")
+            return None
+        query_result = cursor.fetchone()
+        
+        if query_result != None:
+            # Mysql query
+            try:
+                cursor.execute(f"SELECT * FROM {name} WHERE member_id = {message.author.id}")
+            except Error as e:
+                logger.error(f"{e}")
+                return None
+            query_result = cursor.fetchone()
+
+            if query_result != None:
+                # Update messages count
+                try:
+                    cursor.execute(f"UPDATE {name} SET messages_count = %s WHERE member_id = %s", (query_result["messages_count"] + 1, message.author.id))
+                except Error as e:
+                    logger.error(f"{e}")
+                    return None
+                else:
+                    logger.info(f"{message.author.name}({message.author.id}): Got 1 message count in {name}")
+                    mysqldb.commit()
+            else:
+                try:
+                    cursor.execute(f"INSERT INTO {name} (member_id, messages_count) VALUES (%s, %s)", (message.author.id, 1))
+                except Error as e:
+                    logger.error(f"{e}")
+                    return None
+                else:
+                    mysqldb.commit()
+                    logger.warning(f"New member was added to {name} table! ({message.author})")
         else:
-            logger.warning(f"New member was added to topMonths database! ({message.author})")
-            topMonths_col.document().set(
-                {
-                    "member_id": message.author.id,
-                    "messages_count": 1,
-                }
-            )
+            try:
+                cursor.execute(f"CREATE TABLE {name} (`member_id` VARCHAR(18) NOT NULL, `messages_count` INT(10) NOT NULL, `voice_time` INT(10) NOT NULL DEFAULT 0, PRIMARY KEY (`member_id`));")
+            except Error as e:
+                logger.error(f"{e}")
+                return None
+            else:
+                mysqldb.commit()
+                logger.warning(f"New table {name} was added to database!")
+            
+            try:
+                cursor.execute(f"INSERT INTO {name} (member_id, messages_count) VALUES (%s, %s)", (message.author.id, 1))
+            except Error as e:
+                logger.error(f"{e}")
+                return None
+            else:
+                mysqldb.commit()
+                logger.warning(f"New member was added to {name} table! ({message.author})")
+            
             logger.info(f"{message.author.name}({message.author.id}): Got 1 message in month {month}.{year}")
```

Comment:
I get the general idea, but I have a suggestion to create it differently:
Create one table with users, add their message count, set message count to 0 each month, no more infinity amount tables, just one (be careful about some edge cases like 28/29/30/31 days, generally use `datetime` library), turbo mega gamma storage savings